.github/workflows/e2eCJS.yml

```
@@ -0,0 +1,118 @@
+name: E2E Test Suite - CJS
+
+on:
+  workflow_run:
+    workflows:
+      - Nightly Build Develop
+      - Test, Build, and Release
+      - Create and Test Beta Release Branch
+    types:
+      - completed
+
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests in CJS'
+        required: false
+        default: 'develop'
+        type: string
+      debugLwcTests:
+        description: 'Debug LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      runLwcTests:
+        description: 'Run LWC Tests'
+        required: false
+        default: true
+        type: boolean
+      visualforceLsp:
+        description: 'Visualforce LSP'
+        required: false
+        default: true
+        type: boolean
+      vscodeVersion:
+        description: 'VSCode Version'
+        required: false
+        default: '1.85.2'
+        type: string
+      runId:
+        description: 'Run ID of the workflow run that created the vsixes'
+        required: true
+        type: string
+
+jobs:
+  debugLwcTests:
+    if: ${{ inputs.debugLwcTests || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/runE2ETest.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranch || 'develop'  }}
+      testToRun: 'debugLwcTests.e2e.ts'
+      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
+      os: '["macos-latest"]'
+
+  runLwcTests:
+    if: ${{ inputs.runLwcTests || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/runE2ETest.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranch || 'develop'  }}
+      testToRun: 'runLwcTests.e2e.ts'
+      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
+      os: '["macos-latest"]'
+
+  visualforceLsp:
+    if: ${{ inputs.visualforceLsp || (github.event_name == 'workflow_run' && github.event.workflow_run.conclusion == 'success') }}
+    uses: ./.github/workflows/runE2ETest.yml
+    secrets: inherit
+    with:
+      automationBranch: ${{ inputs.automationBranch || 'develop' }}
+      testToRun: 'visualforceLsp.e2e.ts'
+      vscodeVersion: ${{ inputs.vscodeVersion || '1.85.2' }}
+      runId: ${{ inputs.runId || github.event.workflow_run.id }}
+      os: '["windows-latest"]'
+
+  slack_success_notification:
+    if: ${{ success() }}
+    needs: [debugLwcTests, runLwcTests, visualforceLsp]
+    uses: ./.github/workflows/slackNotification.yml
+    secrets: inherit
+    with:
+      title: 'E2E Test Suite - ESM'
```

Also make this change for the failure and cancelled notifications.

```suggestion
      title: 'E2E Test Suite - CJS'
```

---

good catch!